db/schema.rb

```diff
@@ -83,7 +83,7 @@
     t.jsonb "description", default: {}, null: false
     t.index ["age_rating"], name: "index_anime_on_age_rating"
     t.index ["average_rating"], name: "anime_average_rating_idx"
-    t.index ["average_rating"], name: "index_anime_on_wilson_ci", order: { average_rating: :desc }
+    t.index ["average_rating"], name: "index_anime_on_wilson_ci", order: :desc
```

Comment:
@NuckChorris why would this change??

Reply:
New Rails version dumps slightly different, this is fine